spec/langchain/assistants/message_spec.rb

```diff
@@ -1,7 +1,41 @@
 # frozen_string_literal: true
 
-RSpec.describe Langchain::Thread do
-  it "raises an error if messages array contains non-Langchain::Message instance(s)" do
-    expect { described_class.new(messages: [Langchain::Message.new, "foo"]) }.to raise_error(ArgumentError)
+RSpec.describe Langchain::Message do
+  subject { described_class.new }
```

Comment:
Btw, in Rails, `subject` always defaults to `described_class.new` so you don't even have to declare it.